tests/parquet_test.py

```diff
@@ -14,13 +14,15 @@ def read_write_test(dtype):
         ak_arr = ak.randint(0, 2**32, SIZE)
     elif dtype =='uint64':
         ak_arr = ak.randint(0, 2**32, SIZE, dtype=ak.uint64)
+    elif dtype =='bool':
+        ak_arr = ak.randint(0, 2**32, SIZE, dtype=ak.bool)
```

Comment:
for the `bool` case, it might be more representative to do `ak.randint(0, 1, adjusted_size, dtype=ak.bool)`

just because in it's current state, it'll only ever be false when we happen to hit a 0 which seems pretty rare. There are 2 other places in this file where this would need to be updated

Reply:
Ah, I was not aware of this, thanks!